### PR TITLE
Milestone 4: Readme game instructions

### PR DESCRIPTION
**Pull request by gurbuzertunga**
In this Milestone we created a readme file with;

- Game instructions,
- Win conditions explained,
- Live Demo of the project,
- İnstallation Guidelines

### CODE REVIEW

File: readme.MD [hunk 1]
```diff
@@ -1,57 +1,106 @@
...
-## :memo: License
+# The Tic-Tac-Toe
+
+> In this project, we are building a Tic Tac Toe game. The main goal is to put into practice the main concepts of Object Oriented Programming. Particularly, classes, modules and objects and access their attributes.
+
+For this project, we wrote a Ruby executable script main.rb that allows the user to play the game of Tic, Tac, Toe in the Terminal.
+
+## How To Play
+
+'Tic, Tac, Toe' is a traditional game for two players played on a board with 9 squares.
```
Comment (kitpao) [inline readme.MD]: - [x] **OPTIONAL** You don't need to put all those comas in the name Tic Tac Toe :)

File: readme.MD [hunk 2]
```diff
@@ -1,57 +1,106 @@
...
+
+If you intend to download the project, you will need to have Ruby already installed on your machine. For more information on how to install Ruby, follow [this link](https://www.ruby-lang.org/en/downloads/).
+
+### Installation Instructions
+
+To get your own copy of our project simply clone the repository to your local machine.
+
+**Step 1**: Using the Command Line, navigate into the location where you would like to have the repository. Then enter the following line of code:
+
+`git clone https://github.com/gurbuzertunga/tic-tac-toe.git`
```
Comment (kitpao) [inline readme.MD:50]: - [x] **OPTIONAL** You can also use 3 backquotes at the beginning and 3 backquotes at the end to format it this way

```
This is an example for formatting code
```
image

File: readme.MD [hunk 3]
```diff
@@ -1,57 +1,106 @@
...
+
+The aim of the game is to have three of your symbols form a line, be it vertically, horizontally or diagonally, as illustrated in the image below.
+
+![A Winning Combination of Tic, Tac, Toe](.github/winning-screenshot.webp)
+
+In the event that all 9 squares are taken without either player getting three of their symbols in a line, a draw is declared.
+
+### Accessing The Game
+
+- To quickly play the game, follow the link in the [Live Demo section](https://repl.it/@gurbuzertunga/tic-tac-toe#bin/main.rb) and hit the Run button in the header. The game will begin in the window on the right.
```
Comment (kitpao) [inline readme.MD]: - [x] I'm sorry, but this instruction doesn't seem to work properly. Please provide a working link or add configuration on .replit instructions.
**Hint:** For more complex projects with many directories, I would suggest you (and this solution is optional) to use Gitpod, just append `gitpod.io/` to the URL of your repo:
https://gitpod.io/github.com/gurbuzertunga/tic-tac-toe/tree/readme_game_instructions

image

File: readme.MD [hunk 4]
```diff
@@ -1,57 +1,106 @@
...
+
+## Built With
+
+- Ruby
+- VSCode
+- Rubocop
+
+## Live Demo
+
+To quickly see our project in action, visit the [live demo]('#') on repl.it.
```
Comment (kitpao) [inline readme.MD]: - [ ] This link is broken, please put the right link here 
image

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello @gurbuzertunga and @mohapakram 

Your readme is pretty complete, and your sections detailed, I really liked it 👏 

:pencil2: However, in order to be professional, your links and instructions should work, please check inline comments for the details 🔽 
image


**_Note:_** All the **[OPTIONAL]** are **not required** in order to approve your project but are personal suggestions to improve your coding experience

### ♻️ Status: Your project requires some additional changes ♻️
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Comment by Dandush03** [comment]
# Status: :rocket: Project Aproved :rocket: 
# Good luck in your path 
# Happy CODING! :nerd_face: :computer:

**Pull request merged**